ScrollableSegmentedControl/ScrollableSegmentedControl.swift

```diff
@@ -530,7 +539,7 @@ public enum ScrollableSegmentedControlSegmentStyle: Int {
         private func configureConstraints() {
             if let underline = underlineView {
                 underline.translatesAutoresizingMaskIntoConstraints = false
-                underline.heightAnchor.constraint(equalToConstant: 3.0).isActive = true
+                underline.heightAnchor.constraint(equalToConstant: underlineHeight).isActive = true
```

Comment:
This would break if we change the underlineHeight once the control is fully initiated. Would be better to store this constraint into variable such as _underlineHeightConstraint.